include/votca/xtp/eigencuda.h

```diff
@@ -0,0 +1,137 @@
+/*
+ *            Copyright 2009-2019 The VOTCA Development Team
```

Comment:
rename files and put Pipeline and Matrix into separate files

Reply:
done